### PR TITLE
Fable2

### PR DESCRIPTION
**Pull request by OlegZee**
Migration to fable2 and new tools (FAKE 5). Samples are functioning!

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -1,69 +1,68 @@
...
-let nugetProjectFolders = [ (filesInDirMatchingRecursive "*.fsproj" (directoryInfo "./src"))  
-                            (filesInDirMatchingRecursive "*.csproj" (directoryInfo "./src"))
-                          ]
-                          |> Array.concat
-                          |> Seq.map (fun m -> m.Directory.FullName)
+// let nugetProjectFolders = [ (DirectoryInfo.getMatchingFilesRecursive "*.fsproj" (DirectoryInfo.ofPath "./src"))  
+//                             (DirectoryInfo.getMatchingFilesRecursive "*.csproj" (DirectoryInfo.ofPath "./src"))
+//                           ]
+//                           |> Array.concat
+//                           |> Seq.map (fun m -> m.Directory.FullName)
```
Comment (ncthbrt) [inline build.fsx]: Can this go?

File: src/Meta.props [hunk 2]
```diff
@@ -7,6 +7,6 @@
 <RepositoryUrl>https://github.com/ncthbrt/Fable.Websockets.Elmish</RepositoryUrl>
 <PackageTags>fable;fsharp;elmish;websockets;observables</PackageTags>
 <Authors>Nick Cuthbert</Authors>
-<Version>0.1.1</Version>
+<Version>0.2.0</Version>
```
Comment (ncthbrt) [inline src/Meta.props:10]: Feel free to change the Meta here to more accurately reflect contributions

File: build.fsx (hunk 1 above)
Comment (OlegZee) [reply build.fsx]: Dropped this.

File: src/Meta.props (hunk 2 above)
Comment (OlegZee) [reply src/Meta.props:10]: Thank you. My contribution is very small, and I learned new things from your code. That is the motivation point.

**Comment by OlegZee** [comment]
Please approve the PR and push a new build if the things are ok in general.
We could discuss/implement more ideas afterwards.

cc @ncthbrt

**Comment by ncthbrt** [comment]
@OlegZee will publish a new version tomorrow evening.

**Comment by OlegZee** [comment]
@ncthbrt please publish the new version as the evening is coming :)

**Pull request merged**